auto_submit/test/service/validation_service_test.dart

```diff
@@ -44,23 +47,75 @@ void main() {
     pubsub.publish('auto-submit-queue-sub', pullRequest);
     QueryResult queryResult = createQueryResult(flutterRequest);
 
-    await validationService.processPullRequest(config, queryResult, pullRequest, 'test', pubsub);
+    await validationService.processPullRequest(
+        config: config, result: queryResult, messagePullRequest: pullRequest, ackId: 'test', pubsub: pubsub);
 
     expect(githubService.issueComment, isNotNull);
     expect(githubService.labelRemoved, true);
     assert(pubsub.messagesQueue.isEmpty);
   });
 
-  group('shouldProcess pull request', () {
-    test('should process message when autosubmit label exists and pr is open', () async {
+  test('Remove label and post comment when no revert label.', () async {
```

Comment:
Let's create a new group for the revert label logics, and add tests for this group:
1) merge PR & create issue & ack message successfully for a valid revert
2) remove label &  ack message & post comment for a non-valid revert.
3) remove label &  ack message & post comment for a failed revert (due to exception etc).

Reply:
Will do.

Reply:
Number 3 has already been implemented but I will add the other two.

Reply:
Added.